### PR TITLE
show list of case forms in correct order

### PR DESCRIPTION
**Pull request by snopoke**
http://manage.dimagi.com/default.asp?225584
@benrudolph

### CODE REVIEW

File: corehq/sql_accessors/sql_templates/get_case_form_ids.sql [hunk 1]
```diff
@@ -3,20 +3,15 @@ DROP FUNCTION IF EXISTS get_case_form_ids(TEXT);
...
-    ) transactions
-    ORDER BY transactions.server_date;
+    SELECT
+        form_processor_casetransaction.form_id
+    FROM form_processor_casetransaction
+    WHERE form_processor_casetransaction.case_id = p_case_id
+    AND form_processor_casetransaction.revoked = FALSE
+    AND form_processor_casetransaction.form_id IS NOT NULL
+    AND (form_processor_casetransaction.type & type_form) = type_form
+    ORDER BY form_processor_casetransaction.server_date;
```
Comment (snopoke) [inline corehq/sql_accessors/sql_templates/get_case_form_ids.sql]: We now only store one transaction per form so no need to do `distinct`.

**Comment by snopoke** [comment]
:white_check_mark: @dannyroberts

**Comment by czue** [comment]
i couldn't tell - does `ordered=False` actually have different behavior? if not, any reason you made it a param as opposed to just changing the default?

**Comment by czue** [comment]
:+1: to merge by me but i'm curious the answer to my question

**Comment by snopoke** [comment]
It only has different behaviour for sql in which case forms are returned on
whichever order the db returns them (which for a sharded setup is not
anything specific)

**Comment by czue** [comment]
can you point me where in the PR that logic lives? I don't see it

**Comment by snopoke** [comment]
It's not part of this PR since the logic already exists but the param wasn't being passed through: https://github.com/dimagi/commcare-hq/blob/master/corehq/form_processor/backends/sql/dbaccessors.py#L73

**Comment by czue** [comment]
:+1: thanks, that's what i was missing

**Comment by dannyroberts** [comment]
👍

**Pull request merged**